packages/dds/tree/src/simple-tree/treeNodeApi.ts

```diff
@@ -203,8 +206,17 @@ export const treeNodeApi: TreeNodeApi = {
 				return undefined;
 			case 1: {
 				const identifier = flexNode.tryGetField(identifierFieldKeys[0] ?? oob())?.boxedAt(0);
+				if (isMapTreeNode(flexNode) && identifier === undefined) {
```

Comment:
The structure of checks can be simplified a bit:

```ts
if (isMapTreeNode(flexNode)) {
	if (identifer === undefined) {
		throw new UsageError();
	}
	return identifer.value as string;
}
```

Reply:
Merged if statements into one